apps/settings/elements/hotspot.html

```diff
@@ -23,7 +23,7 @@ <h2 data-l10n-id="internetSharing-wifi"></h2>
           <a class="info-item">
             <span data-l10n-id="wifi-name" class="info-item-title"></span>
             <span class="info-item-space"></span>
-            <span class="hotspotSSID info-item-content"></span>
+            <span class="hotspotSSID info-item-content" data-name="tethering.wifi.ssid"></span>
```

Comment:
This is not required as we already have a handler updating the span when SSID changes.